lib/amazon_ssa_support/evm_bucket.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'evm_common'
+
+module AmazonSsaSupport
+  module EvmBucket
+    def self.get(arg_hash)
+      raise ArgumentError, "evm_bucket must be specified." if arg_hash[:evm_bucket].nil?
+
+      evm_bucket_name = arg_hash[:evm_bucket]
+      s3              = arg_hash[:s3] || Aws::S3::Resource.new(region: DEFAULT_REGION)
```

Comment:
We probably need to change this going forward. We should probably get the region from the provider.